tests/testthat/test_ms_drop_var_prefix.R

```diff
@@ -0,0 +1,20 @@
+library(macrosheds)
+library(testthat)
+
+
+test_that('var is character', {
+    expect_type(ms_drop_var_prefix('GN_NO3_N'), 'character')
+    expect_type(ms_drop_var_prefix('pm_sub_surf_permeability_sd'), 'character')
+})
+
+test_that('var is of length 1', {
+    expect_equal(length(ms_drop_var_prefix('GN_NO3_N')), 1)
+})
+
+test_that('var is correct nchar', {
+    expect_equal(nchar(ms_drop_var_prefix('GN_NO3_N')), 2)
+})
+
+test_that("underscore didn't get removed", {
+    expect_no_match(ms_drop_var_prefix('GN_NO3_N'), '_')
+})
```

Comment:
@spencerrhea  @westonslaughter it occurred to me after writing the example tests that i left out a very important set: NULL/NA/0-length behaviors. these won't apply to every function in the MS package, but for any function that could receive a wide range of unintended inputs from a user who isn't paying attention, we should take care to make our return values very intentional. for ms_drop_var_prefix, inputs of `NULL` or `character()` should return an empty character class, i.e. `character(0)`, and NA should return NA. This is in keeping with other string subsetting functions in base R and stringr.